### ISSUE TITLE
[feature] custom User-Agent Options add.

### ISSUE THREAD

**Issue by gy741** [projectdiscovery/nuclei#214]
Hello,

"User-Agent" seems to be fixed.

Occasionally, abnormal "User-Agent" data is blocked by a firewall. Therefore, normal (browser-supplied) data is needed.

For example: 
```
Mozilla/5.0 (Android 4.4; Mobile; rv:41.0) Gecko/41.0 Firefox/41.0
```

I can modify it in the source code, but it would be good to add a control function.

```
GET / HTTP/1.1
Host: google.com
Connection: close
Accept: */*
Accept-Language: en
Connection: close
User-Agent: Nuclei - Open-source project (github.com/projectdiscovery/nuclei
```

Thanks.

**Comment by bauthard** [comment]
Hey @gy741, 

It's already there, you can always overwrite any header with the `-H` flag. 

```
echo http://ob38vcxg53qonv1m5e7rkdhmrdx4lt.burpcollaborator.net | nuclei -t tokens -H 'User-Agent: this-is-my-ageny'
```



![image](https://user-images.githubusercontent.com/8293321/89497358-d3d44380-d7d9-11ea-8948-565e3fa5f4eb.png)

**Comment by gy741** [comment]
Hello @bauthard 

Good. 

The problem has been solved. 

I have one more question. Can I change the "Host" header?

I changed the "-H" option and template but it doesn't work.

For example:

```
$ Echo http://ob38vcxg53qonv1m5e7rkdhmrdx4lt.burpcollaborator.net | nuclei -t tokens/amazon-mws-auth-token-value.yaml -H 'Host: this-is-my-ageny' --debug 
....

GET / HTTP/1.1
Host: ob38vcxg53qonv1m5e7rkdhmrdx4lt.burpcollaborator.net
Connection: close
Accept: */*
Accept-Language: en
Connection: close
User-Agent: Nuclei - Open-source project (github.com/projectdiscovery/nuclei)

[INF] Dumped HTTP response for http://ob38vcxg53qonv1m5e7rkdhmrdx4lt.burpcollaborator.net (amazon-mws-auth-token-value)

HTTP/1.1 200 OK
Content-Length: 53
Content-Type: text/html
Server: Burp Collaborator https://burpcollaborator.net/
X-Collaborator-Version: 4

<html><body>ccwuw0fddpz6jwku55ep12zjjgz</body></html>
[INF] No results found. Happy hacking!

```


```
id: test

info:
  name: test
  author: karas
  severity: medium

requests:
  - method: GET
    headers:
      Host: ""
    path:
      - "{{BaseURL}}/"
```

**Comment by bauthard** [comment]
Thank you for confirmation, about your 2nd question, I will confirm and create a new issue if it doesn't work, closing this for now.
